Jenkinsfile

```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/env groovy
 
-/* `buildPlugin` step provided by: https://github.com/jenkins-infra/pipeline-library */
-// tests skipped because no junit reports generated.
-buildPlugin(failFast: false, tests: [skip: true], configurations: [
-        [ platform: "docker", jdk: "8", jenkins: null ],
-        [ platform: "docker-windows", jdk: "8", jenkins: null ],
-])
+/*
+`buildPlugin` step provided by: https://github.com/jenkins-infra/pipeline-library
+*/
+buildPlugin(
+        failFast: false,
+        tests: [skip: true], // tests skipped because no junit reports generated.
+        useContainerAgent: false, // Set to `false` if you need to use Docker for containerized tests
+        configurations: [
+                [ platform: "linux", jdk: "21", jenkins: null ],
+                [ platform: "windows", jdk: "17", jenkins: null ],
```

Comment:
heh..
I just realized that this is to build the native packages and as such ther eis no java source (or tests) and the JDK is only used to run maven.

Reply:
Haha yes, which is why it was left on 8 for so long. But looks like it's good to keep it current since buildPlugin started acting funny on 8 for windows